Kotlin/app/src/main/java/com/taboola/kotlin/examples/MetaConst.kt

```diff
@@ -5,10 +5,12 @@ class MetaConst {
         const val META_PUBLISHER_NAME = "sdk-tester-meta"
         const val AUDIENCE_NETWORK_APP_ID = "1097593608162039"
         const val AUDIENCE_NETWORK_PLACEMENT_ID = "1097593608162039_1097982098123190"
+        const val AUDIENCE_NETWORK_CAROUSEL_PLACEMENT_ID = "1097593608162039_1288555252399206"
         const val AUDIENCE_NETWORK_APPLICATION_ID_KEY = "audienceNetworkApplicationId"
         const val AUDIENCE_NETWORK_PLACEMENT_ID_KEY = "audienceNetworkPlacementId"
         const val ENABLE_META_DEMAND_DEBUG_KEY = "enableMetaDemandDebug"
         const val DEFAULT_LAYOUT_KEY = "default"
-        const val TEST_LAYOUT_TYPE = "image_link"
+        const val TEST_LAYOUT_AD_TYPE = "image_link"
```

Comment:
@voltfmanSlava - Why did you name this TEST_LAYOUT_AD_TYPE in comparison to TEST_LAYOUT_CAROUSEL_TYPE? If in the future you will have another value for an ad type (I.E. IMG_16_9_APP_INSTALL), this naming won't make sense.
https://developers.facebook.com/docs/audience-network/setting-up/test/test-device/

Reply:
@tomer-br-taboola I can rename it to "TEST_LAYOUT_IMAGE_LINK_TYPE" but as i understand from you comment is when we will need to add IMG_16_9_APP_INSTALL type and it will look like "TEST_LAYOUT_IMG_16_9_APP_INSTALL_TYPE" it doesn't make sense. 
I can call it like "TEST_LAYOUT_SINGLE_IMAGE_TYPE" and for the future if we need to add IMG_16_9_APP_INSTALL we can call it "TEST_LAYOUT_SINGLE_IMAGE_WITH_RATIO_OF_16_ON_9_TYPE", do you agree?

Reply:
@voltfmanSlava - If you look at the link I shared, there is only one type of image ads. Meaning, the ratio is always 16/9. The difference is whether it is an application install or image link. So there is no reason to add the ratio there.